fixtures/cleandb.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../app/models');
```

Comment:
Please move this file in the `test` folder. This folder should contain only fixture files.